### PR TITLE
Display three the same records in BlockInfo tab of worker node WEBUI …

### PR DESCRIPTION
**Pull request by phenixmzy**
…#9518

### CODE REVIEW

**Comment by apc999** [comment]
thanks for the contribution, can you sign the [contribution license agreement](https://cla.alluxio.org/)

**Comment by phenixmzy** [comment]
> 
> 
> thanks for the contribution, can you sign the [contribution license agreement](https://cla.alluxio.org/) @phenixmzy
hi @apc999 
i have already signed, clicked the line  [contribution license agreement](https://cla.alluxio.org/)  always show

> Looks like you've already started the process of signing the Contribution License Agreement, thank you!

and

> Status Pending

i am  request submit pr first time so some processes are unfamiliar,but I am happy to contribute to the community.

**Comment by apc999** [comment]
@calvinjia  can you talk a look at this PR which aims to fix #9518?

File: core/common/src/main/java/alluxio/util/webui/UIFileInfo.java [hunk 1]
```diff
@@ -389,4 +392,11 @@ public String getGroup() {
   public String getMode() {
     return mMode;
   }
+
+  /**
+   * Get BlockId List
+   *
+   * @return the BlockId List in worker node
+   * */
+  public List<Long> getmBlockIds() { return mBlockIds; }
```
Comment (calvinjia) [inline core/common/src/main/java/alluxio/util/webui/UIFileInfo.java]: `getBlockIds`

File: core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerRestServiceHandler.java [hunk 2]
```diff
@@ -343,6 +343,9 @@ public Response getWebUIBlockInfo(@QueryParam("path") String requestPath,
                 uiFileInfo
                     .addBlock(blockMeta.getBlockLocation().tierAlias(), blockId, blockSize, -1);
               }
+              //uiFileInfos.add(uiFileInfo);
```
Comment (calvinjia) [inline core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerRestServiceHandler.java]: Remove this line

**Comment by calvinjia** [comment]
jenkins, test this please.

**Comment by calvinjia** [comment]
@phenixmzy We've merged this as part of #9734 , thanks for the contribution!

**Pull request closed without merging**